### PR TITLE
working in builder design

### PR DESCRIPTION
**Pull request by daniel26897**
currently only the main form was designed

### CODE REVIEW

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs [hunk 1]
```diff
@@ -45,6 +45,7 @@ private void populateOptions()
                 listViewOptions.Items.Add(option);
             }
 
+           // optionsToControl["General"].BackgroundImage = ;
```
Comment (dimabru) [inline Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: Please remove comment

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs [hunk 2]
```diff
@@ -69,5 +70,21 @@ private void buttonSave_Click(object sender, EventArgs e)
             buildOptions.Save();
             this.Close();
         }
+
+        private void splitContainerProperties_Panel2_Paint(object sender, PaintEventArgs e)
```
Comment (dimabru) [inline Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: Empty function

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs [hunk 3]
```diff
@@ -69,5 +70,21 @@ private void buttonSave_Click(object sender, EventArgs e)
...
+        {
+
+        }
+       
+
+        private void pictureBox1_Click_1(object sender, EventArgs e)
```
Comment (dimabru) [inline Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: please change the names to meaningful names

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs [hunk 4]
```diff
@@ -69,5 +70,21 @@ private void buttonSave_Click(object sender, EventArgs e)
...
+        {
+            this.Close();
+        }
+
+        private void pictureBox3_Click(object sender, EventArgs e)
```
Comment (dimabru) [inline Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: Same. change name

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 5]
```diff
@@ -3,12 +3,21 @@
...
 using System.Xml;
+using System.Drawing.Drawing2D;
 
 namespace BuilderApplication.View
 {
+    
     public partial class BMainForm : HelperProject.BaseView.BaseMainForm
     {
+        /*
+         * integers set so the borderless fom could move
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: borderless form*

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 6]
```diff
@@ -92,5 +101,53 @@ private void buttonEdit_Click(object sender, EventArgs e)
             XMLBuild xmlBuild = new XMLBuild(buildPath + "\\" + DefaultInfo.BuildDetailsFileName);
             new BuildEditorView(xmlBuild.tasks, buildName, buildDescription).ShowDialog();
         }
+
+        private void BMainForm_Load(object sender, EventArgs e)
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Empty function

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 7]
```diff
@@ -92,5 +101,53 @@ private void buttonEdit_Click(object sender, EventArgs e)
...
+        {
+
+        }
+
+        private void menuStripMain_ItemClicked(object sender, System.Windows.Forms.ToolStripItemClickedEventArgs e)
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Empty function

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 8]
```diff
@@ -92,5 +101,53 @@ private void buttonEdit_Click(object sender, EventArgs e)
...
+        {
+
+        }
+
+        private void pictureBox1_Click(object sender, EventArgs e)
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Meaningful name

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 9]
```diff
@@ -92,5 +101,53 @@ private void buttonEdit_Click(object sender, EventArgs e)
...
+        {
+            this.Close();
+        }
+
+        private void pictureBox2_Click(object sender, EventArgs e)
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Meaningful name

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs [hunk 10]
```diff
@@ -92,5 +101,53 @@ private void buttonEdit_Click(object sender, EventArgs e)
...
+                this.SetDesktopLocation(MousePosition.X - movX, MousePosition.Y - movY);
+            }
+        }
+
+        private void panelBuilderMainForm_MouseUp(object sender, MouseEventArgs e)
+        {
+            mov = 0;
+        }
+
+        private void buttonCurrentBuilds_Click(object sender, EventArgs e)
```
Comment (dimabru) [inline Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: empty function

**Review by dimabru — CHANGES_REQUESTED** [review]
Good job. Please just remove all unnecessary functions and comments and change the functions I mentioned

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs (hunk 2 above)
Comment (daniel26897) [reply Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: fixed, the function was deleted

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs (hunk 3 above)
Comment (daniel26897) [reply Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: the name was changed

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs (hunk 4 above)
Comment (daniel26897) [reply Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: The name was changed

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 5 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: fixed

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 6 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Deleted

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 7 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Deleted

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 8 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Fixed

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 9 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Fixed

File: Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs (hunk 10 above)
Comment (daniel26897) [reply Installation Builder/BuilderApplication/BuilderView/View/BMainForm.cs]: Fixed

File: Common/CommonTester/BaseView/BaseView/PropertiesView.cs (hunk 1 above)
Comment (daniel26897) [reply Common/CommonTester/BaseView/BaseView/PropertiesView.cs]: Removed

**Pull request merged**